pkg/phases/audit/test.go

```diff
@@ -0,0 +1,144 @@
+package audit
+
+import (
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/flanksource/commons/console"
+	"github.com/moshloop/platform-cli/pkg/platform"
+	v1 "k8s.io/api/core/v1"
+)
+
+const jUnitAuditConfigClass = "AuditConfig"
+const jUnitAuditAPIServerStateClass = "AuditApiServerState"
+const jUnitAuditLogs = "AuditLogsPresent"
+
+//Test k8s auditing functionality.
+func Test(p *platform.Platform, tr *console.TestResults) {
+	ac := p.AuditConfig
+
+	if ac == nil {
+		// A nil AuditConfig is a failure as the implicit default if it is not specified
+		// should be 'disabled'
+		tr.Failf(jUnitAuditConfigClass, "There is no AuditConfig")
+		tr.Done()
```

Comment:
Done is called by the test infra - just return if no more testing is possible
```suggestion
		return
```